src/System/tests/IntegratorTest.cpp

```diff
@@ -35,60 +37,130 @@ bool areVectorsEqual(const T& vector1, const U& vector2, const double& tolerance
     return true;
 }
 
-TEST_CASE("Integrate - Linear system")
+template <typename T, typename U>
+bool areMatricesEqual(const T& matrix1, const U& matrix2, const double& tolerance = 0)
```

Comment:
In Eigen you can use [``isApprox``](https://eigen.tuxfamily.org/dox/classEigen_1_1DenseBase.html#ae8443357b808cd393be1b51974213f9c)

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/70/commits/ff98d9a226054d9449e37d43705f4dfeb1347237